packages/calcite-components/src/components/popover/popover.scss

```diff
@@ -3,6 +3,10 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-popover-background-color: Specifies the background color of the component.
+ * @prop --calcite-popover-border-color: Specifies the border color of the component.
+ * @prop --calcite-popover-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-popover-text-color: Specifies the text color of the component.
  * @prop --calcite-popover-z-index: Sets the z-index value for the component.
  */
```

Comment:
I might expect a `--calcite-popover-shadow` css property here.

Reply:
I don't see any shadow related styling on the component. Can you expand on that?

Reply:
It's coming from floating ui - but I don't think a user would need to know that, so maybe a pattern q of how to set that up. As a user I'd want to just set it at the component level like the other adjustments to border color, corner radius, etc.

Reply:
let's leave the floating-ui tokens seperatly.